src/test/scala/edu/ie3/simona/agent/grid/DBFSAlgorithmSupGridSpec.scala

```diff
@@ -198,23 +216,33 @@ class DBFSAlgorithmSupGridSpec
                     )
                   )
                 ) =>
-            case FinishGridSimulationTrigger(3600) =>
-              // we expect another completion message when the agent is in SimulateGrid again
-              expectMsgPF() {
-                case CompletionMessage(
-                      3,
-                      Some(
-                        Vector(
-                          ScheduleTriggerMessage(ActivityStartTrigger(7200), _)
-                        )
-                      )
-                    ) =>
-                // agent should be in Idle again
-                case x =>
-                  fail(
-                    s"Invalid message received when expecting a completion message for simulate grid after cleanup! Message was $x"
+            // we expect another completion message when the agent is in SimulateGrid again
+            case CompletionMessage(
+                  3,
+                  Some(
+                    Vector(
+                      ScheduleTriggerMessage(ActivityStartTrigger(7200), _)
+                    )
                   )
+                ) =>
+              // agent should be in Idle again
+              resultListener.expectMsgPF() {
+                case powerFlowResultEvent: PowerFlowResultEvent =>
+                  powerFlowResultEvent.nodeResults.headOption match {
+                    case Some(value) =>
+                      value.getvMag().getValue shouldBe 1
+                      value.getvAng().getValue shouldBe 0
+                  }
+
+                  powerFlowResultEvent.lineResults shouldBe empty
+                  powerFlowResultEvent.switchResults shouldBe empty
+                  powerFlowResultEvent.transformer2wResults shouldBe empty
+                  powerFlowResultEvent.transformer3wResults shouldBe empty
+              }
+
+              hsActor.expectMsgPF() { case FinishGridSimulationTrigger(3600) =>
```

Comment:
Here, you can use `expectMsg` instead (you're not making use of the partial function)